vendor/phantomjs/render.js

```diff
@@ -55,7 +55,8 @@
 
         var rootScope = body.injector().get('$rootScope');
         if (!rootScope) {return false;}
-        return rootScope.panelsRendered;
+        var panelsToLoad = window.angular.element('div.panel').length;
```

Comment:
? there is no panel-drop-zone element in the solo panel view
http://play.grafana.org/dashboard-solo/db/graph-styles?from=1473928585116&to=1473932185116&panelId=3&width=1000&height=500&fullscreen